pkg/minikube/bootstrapper/bsutil/kverify/api_server.go

```diff
@@ -202,13 +203,16 @@ func apiServerHealthz(hostname string, port int) (state.State, error) {
 		glog.Infof("stopped: %s: %v", url, err)
 		return state.Stopped, nil
 	}
+
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+
+	glog.Infof("%s returned %d:\n%s", url, resp.StatusCode, body)
 	if resp.StatusCode == http.StatusUnauthorized {
-		glog.Errorf("%s returned code %d (unauthorized). Please ensure that your apiserver authorization settings make sense!", url, resp.StatusCode)
-		return state.Error, nil
+		return state.Error, fmt.Errorf("%s returned code %d (unauthorized). Check your apiserver authorization settings:\n%s", url, resp.StatusCode, body)
```

Comment:
I think there was that we had decided to not return error for this. are u sure this is the right thing to do?